akka-contrib/docs/receive-pipeline.rst

```diff
@@ -16,33 +16,46 @@ Some Possible Use Cases
 
 Interceptors
 ------------
-So how does an interceptor look? Well, Interceptors are defined by decorator functions
-of type :class:`Receive => Receive`, where it gets the inner :class:`Receive` by parameter and
-returns a new :class:`Receive` with the decoration applied.
-Most of the times your decorators will be defined as a regular :class:`Receive` with cases
-for the messages of your interest and at some point delegate on the inner :class:`Receive`
-you get by parameter. We will talk about ignored and unhandled messages later.
+Multiple interceptors can be added to actors that mixin the :class:`ReceivePipeline` trait. 
+These interceptors can be thought of as layers: each outer interceptor advising the 
```

Comment:
I'm not sure we should use the `advice` word, the API does not expose any "advice" in methods/types.
I know it's AOP wording and am familiar with it, but would like to avoid it leaking into the docs, since the API does not use such words.